src/test_graph_travers.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding utf-8 -*-
+import pytest
+
+
+GRAPH = {1: [2, 3],
+         2: [],
+         3: [1]}
+
+GRAPH2 = {1: [2, 3],
```

Comment:
This case looks promising to check for the difference between BFT and DFT, but you are not using it. Expand on this -- how can you use this case to prove the difference between the two?